SpaDES4Dummies.Rmd

```diff
@@ -209,12 +230,16 @@ abundancePlot <- function(sim) {
 }
 ```
 
+"It lookd great, but... How does it know that it should plot them **after** the abundance generator?" you ask, puzzled. Good question sir/madam, 5 points for Griffindor!
+
+####**3.Scheduling events**{#anchor3}
 
-####Scheduling events
+Events *within a module* will occur discretely in a given order, specified using the `scheduleEvent` function. These, such as `abundanceInit`, `abundancePlot` and `abundanceSim` are also sub-functions inside the `doEvent.speciesAbundance` function.  Events *between or among modules* will obey either the internal module's scheduling (if the time is set differently for each event, it will jump from module to module, obeying the specified time order), or the order stated when loading the modules (if the time is the same for all events across modules, it will run all events from the first loaded module, then the same for the second module, and so forth).
 
-Now that we've defined our evetn functions, we need to set the order in which they will happen for this module. For example, we don't want the plotting to occur before the generation of species abundances, so we'll schedule all `abundancePlot` events to occur slightly after the   `abundanceSim` events.
+In other words, the order in which each of the *modules* (and consequently their initialisations) are executed is automatically determined by inter-module dependencies (i.e. module inputs that are the outputs of other modules) - I told you **`SpaDES`** is cool! If there are no inter-module dependencies the order is determined by the other in which the modules are listed in the *global.R* script **and/or** by the way we schedule the events.
```

Comment:
Maybe repetitive? I feel we should keep it short and stick to one way of explaining something, instead of presenting two...